static-analysis/autodispose-error-prone/src/main/java/com/uber/autodispose/errorprone/UseAutoDispose.java

```diff
@@ -114,6 +117,32 @@ public UseAutoDispose(ErrorProneFlags flags) {
     return "https://github.com/uber/AutoDispose/wiki/Error-Prone-Checker";
   }
 
+  /**
+   * Return the lifecycle classes on which to apply the Error-Prone check.
+   *
+   * @param inputClasses the custom scopes defined by user.
+   * @param exclusiveScope whether the custom scopes are exclusive.
+   * @return the classes on which to apply the error-prone check.
+   */
+  private ImmutableSet<String> getClassesWithLifecycle(Optional<ImmutableSet<String>> inputClasses,
```

Comment:
We deal with scopes, not lifecycles. Also looks like this could be static?

```suggestion
  private ImmutableSet<String> getClassesWithScope(Optional<ImmutableSet<String>> inputClasses,
```